Lib/colorsys.py

```diff
@@ -1,11 +1,16 @@
 """Conversion functions between RGB and other color systems.
 
-This modules provides two functions for each color system ABC:
+This modules provides two functions for each color system ABC (or ABCD):
 
   rgb_to_abc(r, g, b) --> a, b, c
   abc_to_rgb(a, b, c) --> r, g, b
+  
+In case of ABCD:
+  
```

Comment:
Remove trailing space

```suggestion

In case of ABCD:

```